### DISCUSSION TITLE
Subtitles / Closed Caption Support

### DISCUSSION THREAD

**Discussion by PsyWarVeteran** [OGSR/OGSR-Engine#342] [Ideas]
Lack of subtitles in SoC is troublesome for many such as people with hearing impairment or those stuck behind a language barrier. Even the localized versions of the game has many untranslated dialogs (Stalker jokes, generic lines, some storyline dialogs etc) and generally poor voice acting and sound design that has missing effects.

My suggestion is, if possible, for OGSR Engine to add some sort of subtitle / closed caption system to the game. It would not only help many people who are unable to get the context but can also allow for easy localization of the game to many different languages.

Thank you.

**Comment by seaz5150** [comment]
Already has been done through scripts: https://www.moddb.com/games/stalker/addons/in-game-cc-and-subtitles-1-00

**Reply by PsyWarVeteran** [reply]
I am aware of that mod but not only it is ancient but more importantly it changes minor details about the game such as being able to make a wish at the Wish Granter, added ambient and NPC sounds or disabling PDA features to stop potential crashes. Also, if such a support to be added, I think the creators would want to use their own code to do so.

**Reply by seaz5150** [reply]
Yeah, just saying this doesn't need to be implemented into the engine and probably won't be. You can just not include those features. I remember someone working on their own subtitle script and posting about it on this git, but I have no idea what the status on that is.

Campfire dialogue, jokes and etc. can be subtitled for a long time, just no one wants to do it.

**Reply by PsyWarVeteran** [reply]
Doesn't need to be implemented according to what / who?
"This isn't needed" doesn't mean anything especially since this suggestion was directed to the developers, not users. I can say the same thing about some of your suggestions such as gunshot sound layering, which compared to the subtitle support is extremely miniscule and unnecessary. Why should that be needed in the engine but not subtitles?

STALKER with proper subtitle support that covers:
 - Generic, greeting, attack, panic, pain lines
 - Campfire dialogs and jokes
 - Untranslated story lines
can be extremely beneficial to the overall experience and I don't see how making a discussion out of this accomplishes anything when again, this was simply a suggestion directed to the developers of this fantastic project.

**Reply by seaz5150** [reply]
Key part here being "into the engine". All of those can be subtitled with the existing scripted system (but somebody has to take the time to translate all of them anyway). Think you're taking it a bit too personal instead of thinking about it) 

In the end you're right I don't have a say on what gets added, just wanted to let you know that the system already exists, but you apparently know that.
